Userland/Utilities/unzip.cpp

```diff
@@ -83,16 +84,11 @@ int main(int argc, char** argv)
     args_parser.add_option(output_directory_path, "Directory to receive the archive content", "output-directory", 'd', "path");
     args_parser.add_option(quiet, "Be less verbose", "quiet", 'q');
     args_parser.add_positional_argument(path, "File to unzip", "path", Core::ArgsParser::Required::Yes);
-    args_parser.parse(argc, argv);
+    args_parser.parse(arguments);
 
     String zip_file_path { path };
 
-    struct stat st;
-    int rc = stat(zip_file_path.characters(), &st);
-    if (rc < 0) {
-        perror("stat");
-        return 1;
-    }
+    struct stat st = TRY(Core::System::stat(zip_file_path.characters()));
```

Comment:
The call to characters() isn't necessary?

Reply:
yep, autoconversion will handle that. Is the `.characters()` in the warnln a few lines down necessary?